### DISCUSSION TITLE
Second Pass prompts

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#2136] [General]
Hi,

so for non SDXL, the second pass does a run without any prompt by default, right?

Is there a way to get the first prompt/negative prompt to also be used by the second pass, maybe a [x] checkbox or something? Because it looks like without the prompt, second pass is "smoothing away" some details.

**Comment by vladmandic** [comment]
That's how it works by default in SDnext - if second prompt is empty, it's going to use first prompt.
But refiner overly smoothing is a general thing with refiner.
